tests/unit/executor_test_base.py

```diff
@@ -120,6 +120,8 @@ def clear_db(self, db):
             name="langchain_embedding", data={}, engine="langchain_embedding"
         )
         db.session.add(r)
+        r = db.Integration(name="auto_ts", data={}, engine="auto_ts")
```

Comment:
I don't see any unit tests. @ZoranPandovski should unit tests be created before accepting a submission? If so, please add some. Otherwise, please remove these two lines.